app/src/main/java/net/squanchy/schedule/view/EventViewHolder.kt

```diff
@@ -0,0 +1,15 @@
+package net.squanchy.schedule.view
+
+import android.support.v7.widget.RecyclerView
+
+import net.squanchy.schedule.domain.view.Event
+
+class EventViewHolder(itemView: EventItemView) : RecyclerView.ViewHolder(itemView) {
+
+    fun updateWith(event: Event, listener: ScheduleViewPagerAdapter.OnEventClickedListener?) {
+        (itemView as EventItemView).updateWith(event)
+        itemView.setOnClickListener {
```

Comment:
This should fit on a single line

Reply:
60add831225bc748c4e226ac260861d8bf63cbff